service_mesh/v2x/upgrading-ossm-to-2-1.adoc

```diff
@@ -0,0 +1,94 @@
+[id="ossm-upgrade-to-2-1"]
+= Upgrading {ProductName} from version 2.0 to version 2.1
+include::modules/ossm-document-attributes.adoc[]
+:context: oossm-upgrading-to-2-1
+
+toc::[]
+
+Upgrading from version 2.0 to 2.1 requires manual steps that migrate your workloads and application to a new instance of {ProductName} running the new version.
+
+.Prerequisites
+
+* You must upgrade to {product-title} 4.7. before you upgrade to {ProductName} 2.0.
+* You must have {ProductName} version 2.1.0 operator. If you selected the *automatic* upgrade path, the operator automatically downloads the latest information. However, there are steps you must take to use the features in {ProductName} version 2.1.
+* You must upgrade from {ProductName} 2.0 to 2.1. You cannot upgrade SMCP 1.1 to 2.1 directly.
+
+[id="ossm-upgrading-upgrade-2-1_{context}"]
+== Upgrading to {ProductName} 2.1
+
+To upgrade {ProductName}, you must create an instance of {ProductName} `ServiceMeshControlPlane` v2 resource in a new namespace. Then, once it's configured, move your microservice applications and workloads from your old mesh to the new service mesh.
+
+.Procedure
+
+. Check your v2 `ServiceMeshControlPlane` resource configuration to make sure it is valid.
++
+.. Run the following command to view your `ServiceMeshControlPlane` resource as a v2 resource.
++
+[source,terminal]
+----
+$ oc get smcp -o yaml
+----
++
+.. Check the `spec.techPreview.errored.message` field in the output for information about any invalid fields.
++
+. Back up your control plane configuration. Switch to the project that contains your `ServiceMeshControlPlane` resource.  In this example, `istio-system` is the name of the control plane project.
++
+[source,terminal]
+----
+$ oc project istio-system
+----
++
+. Update the `.spec.version` field from v2.0 to v2.1.
++
+If you see the following message, then you need to change the existing "Mixer" type to “Istiod” type in the existing Control Plane spec before you update the  `.spec.version` field:
```

Comment:
```suggestion
If you see the following message, change the "Mixer" type to “Istiod” type in the existing Control Plane spec before you update the  `.spec.version` field:
```

Reply:
@rolfedh let's touch base on this when we meet